isis/src/dev/apps/camtest/camtest.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<application name="camtest" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://isis.astrogeology.usgs.gov/Schemas/Application/application.xsd">
+  <brief>Perform basic camera accuracy tests</brief>
+
+  <description>
+      This application is for camera development is not supported for general public use.
```

Comment:
Add **and** between "development" and "is".

I would add ```<strong>``` tags around this sentence